### ISSUE TITLE
Training and Test datasets

### ISSUE THREAD

**Issue by peylnog** [IceClear/StableSR#87]
Thank you so much for this excellent work. Can you open source the training set and test datasets used in the paper.

**Comment by IceClear** [comment]
Hi.
For training sets, you can refer to #24.
For test sets, you may refer to the link in readme.

**Comment by peylnog** [comment]
Thanks for your reply. I have downloaded these test sets and I am curious why you crop them to 128?

**Comment by IceClear** [comment]
The original of the test set can reach 2K or larger. The test time can be huge.

**Comment by peylnog** [comment]
> The original resolution of the test set can reach 2K or larger. The test time can be huge.
Thank you. By the way. Are the RealSR, DRealSR, and DPED-iphone test datasets evaluated after retraining on the corresponding original training datasets, or are they directly tested on these datasets after training on datasets such as DIV2K as mentioned in the paper to evaluate generalization of proposed method?

**Comment by peylnog** [comment]
Thank you. By the way. Are the RealSR, DRealSR, and DPED-iphone test datasets evaluated after retraining on the corresponding original training datasets, or are they directly tested on these datasets after training on datasets such as DIV2K as mentioned in the paper to evaluate generalization of proposed method?

**Comment by IceClear** [comment]
We do not finetune on real-world pair datasets. For details you may refer to our paper.

**Comment by Juan-Chen45** [comment]
Thanks for the great work, I wonder if it is possible to open-source the scripts used for pre-processing testsets (adding degradation etc.)? Thank you. @IceClear

**Comment by ZH3nGsQ** [comment]
> Thanks for your reply. I have downloaded these test sets and I am curious why you crop them to 128?

Could you plz tell me how to download those test datasets? I didn't find link in readme to download test datasets.

**Comment by XZYuann** [comment]
> > Thanks for your reply. I have downloaded these test sets and I am curious why you crop them to 128?
> 
> Could you plz tell me how to download those test datasets? I didn't find link in readme to download test datasets.

Maybe the link is here: https://huggingface.co/datasets/Iceclear/StableSR-TestSets
